LiveSplit.AutoSplitters.xml

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>Sker Ritual</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/mitchell-merry/autosplitters/main/Sker%20Ritual/skerritual.asl</URL>
+            <URL>https://github.com/mitchell-merry/asl-help/blob/issue-13-bandaid/lib/asl-help-sker-ritual-patch</URL>
```

Comment:
(!) I have used a custom patch of asl-help. The reason being this issue: https://github.com/just-ero/asl-help/issues/13

I have a patch that works for this game, but will definitely break other games. So, I can use that for specifically this game while a better patch is found. I don't know enough about how to introspect Unity internals to figure out what is actually happening and fix upstream.